### ISSUE TITLE
Medusa waits for seed nodes indefinitely

### ISSUE THREAD

**Issue by adejanovski** [thelastpickle/cassandra-medusa#134]
When restoring a remote cluster, we'll wait for the seed nodes to be started before we can start the other nodes.
To check this, we run a `nodetool -h <seed_node> statusbinary` command from the other nodes, which will fail if Cassandra restricts JMX access to localhost (which is the default).
We need to test something else such as whether the 9042 is accepting connections through `nc`.

relates to #130

**Comment by adejanovski** [comment]
duplicate of #126
